src/sempy_labs/_labels.py

```
@@ -0,0 +1,124 @@
+        key: [{"artifactId": i} for i in dfI[dfI["Type"] == value]["Id"].tolist()]
+        for key, value in grouped_types.items()
+    }
+
+    # Add generic artifact types
+    artifact_ids = dfI[dfI["Type"].isin(fabric_items)]["Id"].tolist()
+    if artifact_ids:
+        payload["artifacts"] = [{"artifactId": i} for i in artifact_ids]
+
+    prefix = _get_url_prefix()
+
+    response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)